packages/hooks/src/NFT/useNFTViewMethod.ts

```diff
@@ -0,0 +1,63 @@
+// @ts-nocheck
```

Comment:
Created an issue to improve this eventually: https://github.com/curaOS/source/issues/102